client.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <boost/asio.hpp>
+#include <string>
+#include <vector>
+#include "ChatMessage.hpp" 
+
+
+class ChatClient {
+    boost::asio::io_context _ioContext;
+    boost::asio::ip::tcp::socket _clientSocket;
+    std::string _userName; 
+    bool _isRunning = true;
+
+    static const int bufferSize;
```

Comment:
If it's static, the member is shared among all instances of this class, if you didn't intend for this, you don't need it to be static